metadata_manager/ap_src_meta_fetcher.py

```diff
@@ -179,16 +179,18 @@ def __get_build_options_at_commit_from_cache(self,
         self.logger.debug(f"Got value {value} at key {key}")
         return dill.loads(value) if value else None
 
-    def __get_boards_at_commit_from_cache(self, commit_id: str) -> list:
+    def __get_boards_at_commit_from_cache(self, commit_id: str) -> tuple:
```

Comment:
```suggestion
    def __get_boards_at_commit_from_cache(self, commit_id: str) -> tuple[list, list]:
```

I think that works?

... might need to check which version of Python3 that's allowed in , 'though

Reply:
Done. Thanks. It works on python 3.10 which we use in the web app.